plugins/catalog-backend/src/service/createRouter.ts

```diff
@@ -228,6 +231,28 @@ export async function createRouter(
     });
   }
 
+  if (orchestrator) {
+    router.post('/validate-entity', async (req, res) => {
+      const bodySchema = z.object({
+        entity: z.unknown(),
+        location: z.string(),
```

Comment:
I think we should limit this to `url` locations only for now, just to be sure, so a quick `parseLocationRef` + validation.